lib/src/lib/table/models/table-cell.model.ts

```diff
@@ -66,4 +66,8 @@ export interface TableCell<T = any> {
    * Enable a tooltip, displays the cell content in a tooltip
    */
   cellTooltip?: (data: T, cell: TableCell) => string;
+  /**
+   * Options for the cellTooltip if available
+   */
+  cellTooltipOptions?: { tooltipPosition: string };
```

Comment:
The tool tip positions are a fixed list of positions so maybe type it as an enum so no incorrect string can be passed?